src/nodejs/neon-js/native/cli/rpc/getBlock.js

```diff
@@ -55,36 +65,37 @@ if (!program.node) {
   netutil.getNodesByTallest(nodes).then(rankedNodes => {
```

Comment:
are you okay with this being run async? the program keeps running on line 78 and further at the same time as this code (just a heads up)

Reply:
Yep, moving the lines below it to before. Good call.